Source/sources/Castro_sources.cpp

```diff
@@ -508,22 +515,13 @@ Castro::get_react_source_prim(MultiFab& react_src, Real time, Real dt)
 
     int ng = 0;
 
-    // Carries the contribution of all non-reacting source terms to
-    // the conserved state (including the hydro update)
-
-    MultiFab A(grids, dmap, NUM_STATE, ng);
-
-    sum_of_sources(A);
-
-    // Compute the state that has effectively only been updated with advection.
-    // U* = U_old + dt A
+    // before the SDC burn, we stashed the state:
+    //    U* = U_old + dt A
     // where A = -div U + S_hydro
```

Comment:
S_hydro means the non-hydro source terms like gravity?

Reply:
I consider gravity to be a hydro source.  It means all but the reactive sources.

Reply:
OK. I understand this but the comment is probably going to be confusing to anyone who doesn't understand our thinking on this

Reply:
I can change it to S_non-reacting